### DISCUSSION TITLE
1.1.2

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#9578] [General]
*  Increase the limit of open files in the Windows -> Linux 9p server
*  Fix various issues in the localhost relay (almost solves #9508)
*  Fix issue where GPU drivers share would not be present both mount namespaces.
*  Don't create a console window if a console app is executed from wslg.exe (solves #9551)
*  Use parent process' interop server if the one passed by WSL_INTEROP does not ecist
*  Add logic to open an existing vhd if the swap VHD can't be created (solves #9395)
*  Fix various issues in the drvfs initialization logic
*  Fix product name, file version and product version on the package executables (solves #9536)
*  Update Microsoft.WSLg to version 1.0.49: 
    * WSLGd: allow customize log file path
    * rdp fontend/backend/shell: keep compositor sleep until window is created
    * rdp shell: allow optional applist folder by .wslgconfig
    * rdp shell: allow alt+F4 to close app
    * libweston-desktop: xwayland window is not visible upon commit until mouse is moved
    * build: use Ubuntu 20.04 for build verification

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/1.1.2'>1.1.2</a>.</em>

**Comment by bplasmeijer** [comment]
localhost relay not solved #9508 

https://github.com/microsoft/WSL/issues/9508#issuecomment-1419234168

I did revert again to 1.0.3
